src/dmd/hdrgen.d

```diff
@@ -3433,3 +3438,24 @@ extern (C++) const(char)* parametersTypeToChars(Parameters* parameters, int vara
     v.parametersToBuffer(parameters, varargs);
     return buf.extractString();
 }
+
+/** Pretty print function parameters.
+ * Params:
+ *  parameter = parameter to print.
+ *  varargs = kind of varargs, see TypeFunction.varargs.
+ *  fullQual = whether to fully qualify types.
+ * Returns: Null-terminated string representing parameters. */
```

Comment:
Nit: Please conform to the existing way Ddoc comments are formatted in the front end. To wit:
```
/****
 * blah blah
 */
```
not:
/** blah blah
   * blah blah */
```

Reply:
Done for this module.